### PR TITLE
Social sample file pilot changes

### PR DESCRIPTION
**Pull request by emilioward**
# Motivation and Context

Due to the format of the social sample file changing we need to update the request sent to EQ for the new attribute fields

# What has changed

Updated Tests and payload being sent to EQ 

# How to test?

Run unit tests and integration tests

# Links

# Screenshots (if appropriate):

### CODE REVIEW

File: app/eq.py [hunk 1]
```diff
@@ -146,12 +145,12 @@ def __init__(self, case: dict, app: Application):
             raise InvalidEqPayLoad(f"Could not retrieve attributes for case {self._case_id}")
 
         try:
-            self._ru_name = self._sample_attributes["Prem1"]
+            self._ru_name = self._sample_attributes["ADDRESS_LINE1"]
```
Comment (jcox-dev) [inline app/eq.py]: I wonder if this should use the new UPRN value 🤔

Comment (emilioward) [reply app/eq.py]: Sorry bit confused as to what you mean, do you mean removing the country_code and address line for the new UPRN value in the try except?

Comment (jcox-dev) [reply app/eq.py]: No, I mean checking with eQ to see if they'd rather the "ru_name" (or some other field) use the UPRN for traceability. Before, all we had was the address to identify a respondent but I think UPRN is supposed to be the unique string.

Comment (emilioward) [reply app/eq.py]: Myself and Liam are going to have a discussion with EQ about the country code data we're passing over to them I could bring that up or you could come along tomorrow?

Comment (jcox-dev) [reply app/eq.py]: Yeah, let's talk about "country code" and the "ru name" tomorrow 👍

**Comment by damien-lloyd** [comment]
Changes look good, including the addresses used for testing.

**Pull request merged**